### PR TITLE
Evil Neuro x filian collab

### PR DESCRIPTION
**Pull request by cjmaxik**
- [x] more sounds for filian
- [x] free space image
- [x] prompts revision

### CODE REVIEW

**Review by tioopental — APPROVED** [review]
LGTM!

**Comment by tioopental** [comment]
@cjmaxik take a look

**Comment by tioopental** [comment]
Oh, actually new sounds needs normalizing

File: src/prompts/characters/filian.js [hunk 1]
```diff
@@ -1,50 +1,68 @@
...
+  '"STOP!"',
+  'effin dies',
+  'makes strange noise',
+  'W RIZZ',
+  'frick',
+  'barks',
+  'rick rolled',
+  'falls from chair',
+  'camera goes crazy',
+  'falls from ledge',
```
Comment (cjmaxik) [inline src/prompts/characters/filian.js:37]: Only applicable for VR Chat streams

File: src/prompts/characters/filian.js [hunk 2]
```diff
@@ -1,50 +1,68 @@
...
+  'hits the wall',
+  'admits she\'s a fussy\xa0baby',
+
+  // model related
+  'bunny girl model',
+  'police filian model',
+  'kawaii filian model',
+
+  // mentions
+  'mentions projektmelody',
```
Comment (cjmaxik) [inline src/prompts/characters/filian.js:49]: She did not mention Melody last two streams. Stale prompt?

File: src/prompts/characters/filian.js [hunk 3]
```diff
@@ -1,50 +1,68 @@
...
+  'falls from ledge',
+  'VR gear bugs out',
+  'breaks character',
+  'hits the wall',
+  'admits she\'s a fussy\xa0baby',
+
+  // model related
+  'bunny girl model',
+  'police filian model',
+  'kawaii filian model',
```
Comment (cjmaxik) [inline src/prompts/characters/filian.js:46]: Prompts based on appearance should be removed

File: src/prompts/characters/filian.js (hunk 2 above)
Comment (tioopental) [reply src/prompts/characters/filian.js:49]: True

**Review by tioopental — APPROVED** [review]
Ready for filipino roast

**Comment by cjmaxik** [comment]
Can we finalize #85 before this one?

**Pull request merged**